src/ui/QuoteMessageInput/QuoteMessageThumbnail.tsx

```diff
@@ -30,33 +30,21 @@ export default function QuoteMessageThumbnail({ message }: Props): ReactElement
     if (
       message.isFileMessage?.()
       && (
-        isImageMessage(message as FileMessage)
-        || isVideoMessage(message as FileMessage)
+        isImageMessage(message)
+        || isVideoMessage(message)
       )
     ) {
       thumbnailUrl = getMessageFirstFileUrl(message);
     } else if (
       message.isMultipleFilesMessage?.()
       && (
         message.fileInfoList.length > 0
-        && isImageFileInfo((message as MultipleFilesMessage).fileInfoList[0])
+        && isImageFileInfo((message).fileInfoList[0])
```

Comment:
need to check isVideoFileInfo maybe 🌝?

Reply:
Ah no but good similar. MFM supports image including GIF. Video is not supported.